main.py

```diff
@@ -20,21 +20,18 @@
 auth.login_tui("unlockApi")
 logging.debug(auth.__dict__)
 
-r = UnlockRequest(auth, "unlock.update.miui.com", "/api/v3/unlock/userinfo", {
+r = UnlockRequest(auth, "unlock.update.intl.miui.com", "/api/v3/unlock/userinfo", {
   "data":{
     "uid":auth.userid,
     "clientId":"1",
-    "clientVersion":"3.3.827.31",
+    "clientVersion":"3.5.1030.37",
     "language":"en",
     "pcId":hashlib.md5(auth.pcid.encode("utf-8")).hexdigest(),
     "region":"",
 }})
 r.add_nonce()
 data = r.run()
 logging.debug(data)
-if not data["shouldApply"]:
-    logging.error("Xiaomi server says shouldApply false, status %d", data["applyStatus"])
-    input("Press Ctrl-C to cancel, or enter to continue. ")
```

Comment:
Apparently, shouldApply was always false in my case, so I removed It.
- When I couldn't unlock because I need to wait X hours it was false 
- When I finally could retrive my unlock token, it was still false

So I thought it was kind of useless right now

Reply:
Okay